main.go

```diff
@@ -56,11 +55,6 @@ func init() {
 
 func main() {
 	initLogger, _ := getLoggerWithLogLevel("info", "")
-	initLogger.Info("version",
-		"GitVersion", version.GitVersion,
```

Comment:
Yeah we never rebuild the same tag so just the tag should be sufficient to identify the commit..